tests/coreml/test_cml_OneHotEncoderConverter.py

```diff
@@ -5,6 +5,14 @@
 import unittest
 import warnings
 import numpy
+try:
```

Comment:
Can this code snippet be refactored as "Imputer = onnxmltools.sklearn.import_Imputer" to avoid a lot of duplication.

Reply:
No, I need to have coremltools Believe that Imputer is at a place it expects to find it.